src/lib/falcor/routes/articles/by-slug.js

```diff
@@ -110,6 +114,29 @@ export default [
         });
       }),
   },
+  {
+    // Get tag information from article
+    route: "articles['bySlug'][{keys:slugs}]['tags'][{integers:indices}]",
```

Comment:
I would prefer making these functions async as well and using maps and `.flatten` as done here: https://github.com/thegazelle-ad/gazelle-server/blob/04c4038bb806f2e3ddc777c8994acbf29596c6f4/src/lib/falcor/routes/articles/by-slug.js#L19-L58